Core/Distrib.php

```diff
@@ -24,5 +24,5 @@
 #  This copyright notice MUST APPEAR in all copies of the script!
 #################################################################
 
-define("BAIKAL_VERSION", "0.5.0");
+define("BAIKAL_VERSION", "0.5.1");
```

Comment:
I don't think we ever released 0.5.0, so maybe just keep the existing version

Reply:
I think that many users have cloned the repo already, even without a release. If we do not bump the version, their upgrade will fail.

Reply:
Cool, sounds good!